biscuit-auth/src/datalog/mod.rs

```diff
@@ -986,19 +1005,21 @@ mod tests {
         w.run(&syms).unwrap();
 
         println!("parents:");
-        let res = w.query_rule(
-            rule::<Term, Predicate>(
-                parent,
-                &[var(&mut syms, "parent"), var(&mut syms, "child")],
-                &[pred(
+        let res = w
+            .query_rule(
+                rule::<Term, Predicate>(
                     parent,
                     &[var(&mut syms, "parent"), var(&mut syms, "child")],
-                )],
-            ),
-            0,
-            &[0].iter().collect(),
-            &syms,
-        ).unwrap();
+                    &[pred(
+                        parent,
+                        &[var(&mut syms, "parent"), var(&mut syms, "child")],
+                    )],
+                ),
+                0,
+                &[0].iter().collect(),
+                &syms,
+            )
+            .unwrap();
```

Comment:
That's why I prefer to move tests in separate files :)
I've clicked 5 times on "Expand all" button to understand we are in a test fn.

Reply:
yeah, tests could be moved to a separate file at some point. not the topic of this pr tho